tests/lib_test.rs

```diff
@@ -165,8 +191,381 @@ fn testnet_verifiable_credential_signing_key() {
+                    genesis_string: "Concordium Testnet Version 5".to_string(),
+                },
+                timestamp: 1709028205,
+                id_cred_sec_hex: "170cd1c77e6f422f6591dd216a4658617908469d22e0b11db185385dd0962114".to_string(),
+            }
+        ).unwrap();
+
+    // The result is almost entirely non-reproducible,
+    // so the test only verifies that the result was computed successfully and that result has the correct JSON format.
+    serde_json::from_str::<IdentityRecoveryRequestResult>(&res_json).unwrap();
+}
+
+/* Account credential creation and serialization tests */
+
+#[test]
+fn account_credential_deployment_concordium_ip() {
+    let res = account_credential(AccountCredentialParameters {
```

Comment:
The blobs were created with the example wallet in the Java SDK because I didn't know the proper types initially. I agree that they're hideous and would like to do a followup to generate them properly instead. But for now I need to proceed with the SDK :)